.pipelines/daily_addons_enablement_test.yaml

```diff
@@ -0,0 +1,56 @@
+schedules:
+- cron: '0 0 * * *'
+  displayName: Daily midnight test
+  branches:
+    include:
+    - ci_prod
+jobs:
+    - job: test
+      variables:
+        armServiceConnectionName: 'ci-1es-acr-connection'
+        subscription: '9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb'
+      pool:
+        name: Azure-Pipelines-CI-Test-EO
+      steps:
+          - task: AzureCLI@2
+            inputs:
+              azureSubscription: '${{ variables.armServiceConnectionName }}'
+              scriptType: 'bash'
+              scriptLocation: 'inlineScript'
+              inlineScript: |
+                az --version
+                az account show
+                az account set -s ${{ variables.subscription }}
+                
+                # Define the resource group and AKS cluster name
+                resourceGroup="ci-logs-prod-aks"

```

Comment:
use the pipeline variables for all the internal specific aksclusterName, logAnalyticsworkspace etc. and instead of adding these in to github.